homework3/.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
.gitignore should be in a core folder of the project. you do not need it in every folder